Riot/Modules/Contacts/ContactsTableViewController.m

```diff
@@ -84,6 +84,9 @@ - (void)viewDidLoad
         [[[self class] nib] instantiateWithOwner:self options:nil];
     }
     
+    // Load the local contacts for display
+    [self refreshLocalContacts];
```

Comment:
As `[CNContactStore authorizationStatusForEntityType:CNEntityTypeContacts]` can trigger permission alert. It can be more appropriate to move this method in `viewDidAppear`. If `viewDidAppear` happen too often or too many refreshes are performed, we can improve later the refresh by setting a rate limit or add checks to reduce occurrences.